modules/restful_example/src/Plugin/resource/Articles__1_0.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\restful_example\Plugin\resource\Articles__1_0.
+ */
+
+namespace Drupal\restful_example\Plugin\resource;
+
+use Drupal\restful\Plugin\resource\ResourceEntity;
+use Drupal\restful\Plugin\resource\ResourceInterface;
+
+/**
+ * Class Articles
+ * @package Drupal\restful\Plugin\formatter
+ *
+ * @Resource(
+ *   name = "articles:1.0",
+ *   resource = "articles",
+ *   label = "Articles",
+ *   description = "Export the article content type with cookie authentication.",
+ *   authenticationTypes = TRUE,
+ *   authenticationOptional = TRUE,
+ *   dataProvider = {
+ *     "entityType": "node",
+ *     "bundles": {
```

Comment:
Cool! I'm leaving TODOs scattered with the new things that we will need to document. This shall be one of them too.